README.md

```diff
@@ -132,6 +132,7 @@ See code for all available configurations.
 | [Framework 11th Gen Intel Core](framework)                          | `<nixos-hardware/framework>`                       |
 | [Framework 12th Gen Intel Core](framework/12th-gen-intel)           | `<nixos-hardware/framework/12th-gen-intel>`        |
 | [Framework 13th Gen Intel Core](framework/13th-gen-intel)           | `<nixos-hardware/framework/13th-gen-intel>`        |
+| [Framework AMD Ryzen 7040 Series](framework/7040-amd)               | `<nixos-hardware/framework/7040-amd>`              |
```

Comment:
I thought about this a bit and I think it's better if distingish between Framework Laptop 16 and Framework Laptop 13. Even it turns out to be identical its better to have different models in the exposed API. We can always internally point to the same configuration. Does this make sense? I made this mistake with the 11th gen, and now we have no way to separate it from the generic configuration.

Reply:
I totally agree. Are there any possibilities to restructure the directory structure (I think there is not "the solution" so far)?
Maybe backwards comptability for some time and add a warning?

I think it would be confusing if only the 13 inch AMD model has a 13 inch path and the others only have the old paths.

Reply:
Do we need a 13-inch part. Is there an overlap in the model numbers?

Reply:
There are no real model numbers as you can see, only a descriptive name for the generation. The 16-inch version currently preparing to ship (aka the only one so far) is officially known as “Framework Laptop *16* (AMD Ryzen 7040 Series)”, while one of the 13-inch ones listed here is “Framework Laptop *13* (AMD Ryzen 7040 Series)”. So unless we want to go with motherboard part numbers instead (and those seem too granular unfortunately), I’d say we do. I mean, the 16-inch does have 7x40HSs where the 13-inch has 7x40Us, but identifying them with that feels wonky.

Reply:
I totally agree with all of the above - I made changes in line with @leona-ya 's original suggestion to put it in a 13-inch folder and updated the linking modules to reflect that.

I took the liberty of creating an issue #773 to hopefully spin off the conversation, assuming the new folder is acceptable? I don't want to leave this PR in a murky state implementing both changes and prolong the review process if I can help it